index-handlers/src/main/java/no/sikt/nva/nvi/index/apigateway/SearchNviCandidatesHandler.java

```diff
@@ -110,6 +107,29 @@ private static boolean userIsNotNviAdmin(RequestInfo requestInfo) {
         return !userIsNviAdmin(requestInfo);
     }
 
+    private List<String> getQueryParamAffiliationsOrViewingScope(RequestInfo requestInfo)
```

Comment:
Can this be extracted in CandidateSearchParameters.fromRequestInfo() directly?
It seems like all others params are read from this method

Reply:
I considered it, but it required an `IdentityServiceClient`-dependency in `CandidateSearchParameters` to fetch the viewingScope. Therefore, decided to keep it out of  `CandidateSearchParameters`